test/test_file_processor.py

```diff
@@ -65,3 +65,49 @@ def test_load_csv_with_cp932(self):
                 # read with cp932 to check if the file is dumped with cp932
                 loaded_df = processor.load(f)
                 pd.testing.assert_frame_equal(df, loaded_df)
+
+
+class TestFeatherFileProcessor(unittest.TestCase):
+    def test_feather_should_return_same_dataframe(self):
+        df = pd.DataFrame({'あ': [1, 2, 3], 'い': [4, 5, 6]})
```

Comment:
[question][nits]

any reason you use hiragana here ?

it doesn't depends on it's hiragana/ascii right ? then ascii sounds fine (to avoid question that hiragana is important or not)